### PR TITLE
Fix/5133/update borders

### LINKED ISSUE #5133: Ensure all narrow content views in fullpage views have border
Since we changed the background color for fullpage views, all "containerized" views require borders

Image
Image
Image

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out Leather build 5dffaf7 — [Extension build](https://github.com/leather-wallet/extension/actions/runs/8570519229), [Test report](https://leather-wallet.github.io/playwright-reports/fix/5133/update-borders), [Storybook](https://fix-5133-update-borders--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=fix/5133/update-borders)

This PR:
- adds borders to the `<Page` component so all pages have the required border
- refactors some other cards to deprecate `<InfoCardFooter` and standardise UI
- adds a fix for the BG location of the send ordinal flow dialog

### CODE REVIEW

**Comment by pete-watters** [comment]
I moved this back to draft as I need to do some more testing and I will also deprecate `InfoCardFooter` once and for all

File: src/app/pages/send/send-crypto-asset-form/form/brc-20/brc20-send-form-confirmation.tsx [hunk 1]
```diff
@@ -103,34 +104,39 @@ export function Brc20SendFormConfirmation() {
...
+      data-testid={SendCryptoAssetSelectors.ConfirmationDetails}
+      footer={
+        <Footer variant="card">
+          <Button aria-busy={isBroadcasting} onClick={initiateTransaction} width="100%">
+            Create transfer inscription
+          </Button>
+        </Footer>
+      }
+    >
+      <CardContent p={0}>
```
Comment (fbwoolf) [inline src/app/pages/send/send-crypto-asset-form/form/brc-20/brc20-send-form-confirmation.tsx]: "space.00"?

**Review by fbwoolf — APPROVED** [review]
Might be worth revisiting the card stories? I'm not sure what the footer `variant="card"` means?

**Comment by pete-watters** [comment]
> Might be worth revisiting the card stories? I'm not sure what the footer `variant="card"` means?

You're right, I will update the stories and add better documentation to make it clearer.  

Footer `variant='card'` is used when the `<Footer` needs to be used inside of a `Card`. 

```
   <Card
      ...
        footer={
          <Footer variant="card">
            <Button
              Continue
            </Button>
          </Footer>
        }
      >
```

In those cases it needs to be positioned differently based on the browser width:
```
export function Footer({ children, variant = 'page', flexDirection = 'column' }: FooterProps) {
  return (
    <styled.footer
     ....
      maxWidth={variant === 'card' ? 'pageWidth' : '100%'}
      position={variant === 'card' ? ' absolute' : 'fixed'}
      ...
      bg={variant === 'page' ? 'ink.background-primary' : undefined}
      borderTop={variant === 'page' ? ' default' : 'none'}
    >
    ...
    </styled.footer>
  );
}
```

The only other footer variant is `page` which it defaults to. I can update it so that `page` is set explicitly elsewhere? 

The concept is that sometimes we have a `<Page` container with a footer and other times we have a `<Card` which is sometimes nested in a page. 

Here is an example of what it's doing on the `Swap` page: 

https://github.com/leather-wallet/extension/assets/2938440/7c063140-48f9-4543-bf98-f4d82998c7a5

**Comment by pete-watters** [comment]
@fbwoolf - I had to switch from working on this to fixing [another bug](https://github.com/leather-wallet/extension/issues/5149) I had introduced. 

I was going to update storybook in here but decided to take that into a different branch and merge this first. The storybook update branch is - `fix/5133/update-container-storybook`, which I pushed it just in case. I'll clean up all the stories for containers and make it easier to understand how I have set things up.

**Pull request merged**